www/js/survey/enketo-survey-service.js

```diff
@@ -64,6 +64,15 @@ angular.module('emission.enketo-survey.service', [
         null;
   }
 
+  function getFirstUserProfile(user_properties, answers) {
+    const potentialCandidates = answers.filter(function(answer) {
```

Comment:
why do you need to do this? you should not be able to retrieve profiles for other users anyway. You can argue that it doesn't hurt anything, but extra code = extra computation = slower response time.